app/bundles/LeadBundle/Resources/views/List/form.html.twig

```diff
@@ -72,17 +72,20 @@
                                 <select class="chosen form-control" id="available_segment_filters">
                                     <option value=""></option>
                                     {% for object, field in form.vars.fields %}
-                                      {% set icon = 'company' == object ? 'building' : 'user' %}
                                       {% set header = translatorHasId('mautic.lead.' ~ object) ? ('mautic.lead.' ~ object)|trans : object|trans %}
                                       <optgroup label="{{ header }}">
                                           {% for value, params in field %}
                                               {% set operators = params.operators is not empty ? params.operators|json_encode() : '{}' %}
+                                              {% set objectLable = translatorHasId('mautic.lead_list.filter.' ~ object) ? ('mautic.lead_list.filter.' ~ object)|trans : object|trans  %}
```

Comment:
Just a small typo in `objectLable` variable

Reply:
Fixed typo. Thanks.